### ISSUE TITLE
An error occurred while trying to create the ticket.

### ISSUE THREAD

**Issue by abais90** [opensupports/opensupports#260]
Hello,

Im unable to create tickets, im getting errors straight away. All SMTP ports have been set.
Anybody familiar with this error?

Please help

**Comment by svoop** [comment]
Same here.

**Comment by svoop** [comment]
@ivandiazwm I'm having the same issue on a brand new install of 4.2.0. Could this be related to #289?

**Comment by svoop** [comment]
Hmmm, shouldn't there be a "tickets" table? The installation wizard didn't set up any...

<img width="942" alt="screen shot 2018-09-20 at 23 03 26" src="https://user-images.githubusercontent.com/7255/45846922-8061c500-bd29-11e8-9aa3-a78776b9b37a.png">

**Comment by ivandiazwm** [comment]
`ticket` table creates automatically when the first ticket is created (the ORM we use, redbean manages it automatically)

@svoop Are you using a user system in your installation?

**Comment by svoop** [comment]
@ivandiazwm Nope, the user system is deactivated.

**Comment by ivandiazwm** [comment]
I think I was able to reproduce the issue.
@svoop You're trying to create it from the admin panel, right?

**Comment by svoop** [comment]
@ivandiazwm It doesn't work neither way: Not in the admin panel, nor via the "create ticket" frontend (in a different browser with no session).

**Comment by ivandiazwm** [comment]
What does the browser console logs when you try to create a ticket? Could you share the result of the request?

**Comment by svoop** [comment]
Good call, apparently, my PHP config is not up to the task:

```
file_get_contents(): https:// wrapper is disabled in the server configuration by allow_url_fopen=0
```

**Comment by svoop** [comment]
With `allow_url_fopen`, the ticket submission works just fine.

However, this setting is an unnecessary security risk. It's maybe worth replacing `file_get_contents` with `curl`.

**Comment by instanttim** [comment]
I'm having this problem too. I can't seem to find any logging or otherwise that would help with diagnosing the specific problem. I was able to create one ticket, but now i can't make any more.

**Comment by instanttim** [comment]
after a lot of wrangling, and reinstalling, it seems to be working more reliably now.

**Comment by instanttim** [comment]
I'm having this issue as well actually. Along with existing tickets not being editable.

javascript console is saying "Unhandled Promise Rejection [object Object]"

**Comment by lyrics1988123** [comment]
im having the same issue with newest version.. 
where to i activate the allow_url_fopen to fix it?

**Comment by agcarlosandres** [comment]
Solucione este problema generando permisos 777 en la carpeta /api/ y sus subdirectorios.

**Comment by svoop** [comment]
@lyrics1988123 `allow_url_fopen` is a PHP setting, so (in order of intrusiveness) set it either in:

* `.htaccess` (Apache)
* webserver config
* `php.ini`

**Comment by lyrics1988123** [comment]
thank you, but already fixed

**Comment by filipemurias** [comment]
I had this issue with version 4.4.0 and solved it by changing the permissions of api/vendor/ezyang/htmlpurifier/library/HTMLPurifier/DefinitionCache/Serializer/HTML/ to 777.

**Comment by intruder214** [comment]
@lyrics1988123 how did u fix the issue regarding creation of ticket with error ?

**Comment by lyrics1988123** [comment]
sadly i dont know it anymore.. im dumb that i didnt post it here.. sry

**Comment by igpf** [comment]
hi guys,i tried allow_url_fopen = On, and api/vendor/ezyang/htmlpurifier/library/HTMLPurifier/DefinitionCache/Serializer/HTML/ to 777.

i still can't create tickets? any thoughts? i'm running version 4.4.0, clean install with recaptcha v2 ??

**Comment by cracarlos** [comment]
@igpf Hola! Lo solucione haciendo chmod 777 api/vendor/ezyang/htmlpurifier/library/HTMLPurifier/DefinitionCache/Serializer /

Espero que te sea de ayuda.

**Comment by aitecc** [comment]
Hallo everyone, same issue there with version 4.5.0 .
allow_url_fopen = On
chmod 777 on all the Serializer folder and files but no luck .

I can't create / edit tickets, and I don't see any statistic (screen remains blank). 
All the other stuff are working as expected.

If someone have any idea / suggestion they're welcome.

**Comment by thecrmsolutions** [comment]
Just installed fresh version and getting "An error occurred while trying to create the ticket." when I add another ticket from the same user.

**Comment by curtive** [comment]
I am not quite sure **where** the error lies, but it seems to be an CHMOD issue. I fixed this issue by chmod 777 -> /usr/local/lsws/raining/html/ticket/api/vendor/ezyang **all folders**.

**Comment by thecrmsolutions** [comment]
@curtive, yes I tried that as well. Then I  saw another post where users had the same issue when installing opensupports via Softaculous. So I removed my installation, downloaded the package from the website and did a custom installation. And then everything worked just fine.

**Comment by DigitalSpaceLabs** [comment]
> @curtive, yes I tried that as well. Then I saw another post where users had the same issue when installing opensupports via Softaculous. So I removed my installation, downloaded the package from the website and did a custom installation. And then everything worked just fine.

This is the perfect solution! It worked like a charm for me. Looks like the installer available pn Softaculous is having issue which is resulting in this error message.

I was having same error message issue when trying to create a new ticket either from admin or client mode. But after installing the application manually as suggested above, it solved the issue.

PS: I created my account on GitHub to specifically recommend this solution, and comment on it :)
